app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -413,4 +413,132 @@
   .media-player {
     @include media-player;
   }
+
+  // This block is duplicated from Whitehall as a transitional step, see the
+  // commit adding this warning before making changes, original version:
+  // https://github.com/alphagov/whitehall/blob/master/app/assets/stylesheets/frontend/helpers/_attachment.scss
+  $thumbnail-width: 99px;
+
+  .attachment {
+    position: relative;
+    margin: $gutter 0;
+    padding: $gutter-half 0 0 ($thumbnail-width + $gutter);
+    @extend %contain-floats;
+
+    &.inline {
```

Comment:
Seems crazy for the inline version to inherit from `.attachment` when it's effectively just a link, and these styles are resetting everything just set.